backend/v2/lumens.js

```diff
@@ -60,6 +68,8 @@ function updateApiLumens() {
       totalSupplyData = response.totalSupply * 1;
       circulatingSupplyData = response.circulatingSupply * 1;
 
+      totalSupplySumData = totalSupplySum * 1;
```

Comment:
❔ What's the significance of the `* 1`?

Reply:
Is this possibly type coercion? I see this in a couple places in this repo

Reply:
I missed that that's actually converting the values to number when storing it. It actually loses a decimal point. Changed to keep as string here:
https://github.com/stellar/dashboard/pull/182/commits/a9d56abdb4f8558e7747f5be6bb23160c2f1bae3

@accordeiro these two endpoints are now returned as string instead of number to save precision. Looks like they're for CoinMarketCap. Is this something we should notify them about?
```
/* For CoinMarketCap */
app.get("/api/v2/lumens/total-supply", lumensV2.totalSupplyHandler);
app.get("/api/v2/lumens/circulating-supply", lumensV2.circulatingSupplyHandler);
```

Reply:
Yes, I think we should notify. But to prevent any breakage in the meantime, instead of replacing fields, can we just add **new** fields with correct precision values?

Reply:
Or a `/v3` API namespace

Reply:
A `/v3` API namespace would allow us to track usage.

Reply:
sgtm
https://github.com/stellar/dashboard/pull/182/commits/d072a589c5f1951acf7e842c3a2cd7f778711738